warmup.py

```diff
@@ -240,17 +251,39 @@ def accumulate(self, L, points):
         The sample is accumulated into the radial distribution average.
 
         """
-        pass
+        if self.rmax >= L/2:
+            raise ValueError('rmax >= L/2')
+        temp_rdf = np.zeros_like(self._rdf)
```

Comment:
Since `points` can come in any form (lists, tuples, etc.), you might want to sanitize it into a numpy array using `np.atleast_2d`. This will ensure that it has a 2d shape.